spec/system/admin/bulk_order_management_spec.rb

```diff
@@ -526,7 +532,8 @@
           visit_bulk_order_management
         end
 
-        it "displays a select box for distributors, which filters line items by the selected distributor", retry: 3 do
+        it "displays a select box for distributors, which filters line items "\
+           "by the selected distributor", retry: 3 do
```

Comment:
Ideally we'd make our example names more concise when we write them in the first place..